cli.go

```diff
@@ -208,5 +211,23 @@ func (app *TGFApplication) Run() int {
 		Printf("tgf v%s\n", version)
 		return 0
 	}
+
+	if !app.NoUpdate {
+		didUpdate := RunUpdate()
+
+		if didUpdate {
```

Comment:
I would add a refresh cooldown period here. Have a local file that prevents updating more than once every 2 hours, let's say. We currently do this with docker images. See `touch.go`